### PR TITLE
21.2 SQL Feature Support (SQL XP)

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/12148.

I've gone through [PostgreSQL Compatibility](https://www.cockroachlabs.com/docs/v21.2/postgresql-compatibility.html) and [SQL Feature Support](https://www.cockroachlabs.com/docs/v21.2/sql-feature-support.html), and I believe this PR finishes updating our SQL Feature Support for 21.2, following https://github.com/cockroachdb/docs/pull/12211. 

LMK if you disagree though. Happy to make edits/additions!

### CODE REVIEW

File: v21.2/sql-feature-support.md [hunk 1]
```diff
@@ -168,16 +168,19 @@ table tr td:nth-child(2) {
...
  Privileges | ✓ | Standard | [Privileges documentation](authorization.html#assign-privileges)
+ Default privileges | ✓ | PostgreSQL Extension | [Default privileges documentation](authorization.html#default-privileges)
 
 ### Miscellaneous
 
  Component | Supported | Type | Details
 -----------|-----------|------|---------
  Column families | ✓ | CockroachDB Extension | [Column Families documentation](column-families.html)
  Computed columns (stored and virtual) | ✓ | Common Extension | [Computed Columns documentation](computed-columns.html)
+ `ON UPDATE` expressions | ✓ | PostgreSQL, MySQL Extension | [`ON UPDATE` expressions documentation](create-table.html#on-update-expressions)
```
Comment (vy-ton) [inline v21.2/sql-feature-support.md:179]: This is only MySQL

File: v21.2/sql-feature-support.md [hunk 2]
```diff
@@ -168,16 +168,19 @@ table tr td:nth-child(2) {
  Roles | ✓ | Standard | [Roles documentation](authorization.html#roles)
  Object ownership | ✓ | Common Extension | [Ownership documentation](authorization.html#object-ownership)
  Privileges | ✓ | Standard | [Privileges documentation](authorization.html#assign-privileges)
+ Default privileges | ✓ | PostgreSQL Extension | [Default privileges documentation](authorization.html#default-privileges)
```
Comment (vy-ton) [inline v21.2/sql-feature-support.md:171]: I would leave this as partial support for now

**Review by vy-ton — APPROVED** [review]

**Comment by vy-ton** [comment]
In https://github.com/cockroachdb/docs/issues/12148#issuecomment-963508558, I also mentioned that we should change full-text indexes to unsupported. Planned could be misleading

File: v21.2/sql-feature-support.md (hunk 2 above)
Comment (ericharmeling) [reply v21.2/sql-feature-support.md]: Done

**Comment by ericharmeling** [comment]
> In [#12148 (comment)](https://github.com/cockroachdb/docs/issues/12148#issuecomment-963508558), I also mentioned that we should change full-text indexes to unsupported. Planned could be misleading

This was already done in https://github.com/cockroachdb/docs/pull/12211/files#diff-b5e637861839995a38065312504994add6450632f5c0fd2a7ab5896200ae3a14R86

File: v21.2/sql-feature-support.md (hunk 1 above)
Comment (ericharmeling) [reply v21.2/sql-feature-support.md]: Done

**Review by ericharmeling — COMMENTED** [review]
TFTR!

File: v21.2/sql-feature-support.md [hunk 3]
```diff
@@ -168,16 +168,19 @@ table tr td:nth-child(2) {
  Roles | ✓ | Standard | [Roles documentation](authorization.html#roles)
  Object ownership | ✓ | Common Extension | [Ownership documentation](authorization.html#object-ownership)
  Privileges | ✓ | Standard | [Privileges documentation](authorization.html#assign-privileges)
+ Default privileges | Partial | PostgreSQL Extension | [Default privileges documentation](authorization.html#default-privileges)
 
 ### Miscellaneous
 
  Component | Supported | Type | Details
 -----------|-----------|------|---------
  Column families | ✓ | CockroachDB Extension | [Column Families documentation](column-families.html)
```
Comment (stbof) [inline v21.2/sql-feature-support.md:177]:
```suggestion
 Column families | ✓ | CockroachDB Extension | [Column families documentation](column-families.html)
```

File: v21.2/sql-feature-support.md [hunk 4]
```diff
@@ -168,16 +168,19 @@ table tr td:nth-child(2) {
...
  Computed columns (stored and virtual) | ✓ | Common Extension | [Computed Columns documentation](computed-columns.html)
+ `ON UPDATE` expressions | ✓ | MySQL Extension | [`ON UPDATE` expressions documentation](create-table.html#on-update-expressions)
  Multi-region capabilities | ✓ | CockroachDB Extension | [Multi-region documentation](multiregion-overview.html)
```
Comment (stbof) [inline v21.2/sql-feature-support.md:180]: It's a bit weird that we use title case for links to titles, and sentence case for links to headings, but I guess that's our convention.

**Review by stbof — APPROVED** [review]

**Pull request merged**